### ISSUE TITLE
[SUGGESTION] Give or drop effect enhancement

### ISSUE THREAD

**Issue by cheeezburga** [ShaneBeee/SkBee#532]
### Suggestion

This is actually in reference to the effect this time. Was wondering if SkBee would be open to the syntax for the give or drop effect being something like: `give or drop %itemtypes% to %players% [owner:(preserving|with) ownership [time:for %-timespan%]]`.

Pretty telling in the pattern, but it would allow players to specify if they want the items that were dropped to only be able to be picked up by the player who was supposed to receive them. Additionally, the time tag could be used if they wanted the dropped items to be pickupable but only after a certain time.

Not super sure if this would be something SkBee would add, cause I feel like you'd have to listen to all the pickups then, and check the nbt for the owner and timespan (at least thats how I'm doing it in Skript at the moment).

### Alternatives

Got a custom effect in skript-reflect which does this, but if SkBee was open to adding this as the official syntax, I think it'd be a welcome addition.

**Comment by cheeezburga** [comment]
Yeah can't argue with that really. As I was writing it I was thinking more and more that it probably didn't really fit into SkBee as well as I had previously thought.

**Comment by ShaneBeee** [comment]
Personally I think this is a lot to throw into one effect.
As Fuse said, this doesn't really mix in well.

That said, with the new expression, you can get the returned items, drop them yourself and modify the dropped entity as you wish.
